index.html

```diff
@@ -210,7 +210,76 @@ <h4>Skills</h4>
             </ul>
           </section>
         </div>
-    </section>
+      </section>
+      <!--Contact form-->
+      <section class="contact">
+        <h2 class="contact-message">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+        <form
+          action="https://formspree.io/f/mnqrnjnk"
+          method="post"
+          class="contact-form"
+        >
+          <div class="form-element">
+            <input
+              type="text"
+              name="name"
+              id="name"
+              required
+              placeholder=" Full Name"
+              maxlength="30"
+            />
+          </div>
+          <div class="form-element">
+            <input
+              type="email"
+              name="email"
+              id="email"
+              required
+              placeholder=" Email address"
+            />
+          </div>
+          <div class="form-element">
+            <textarea
+              name="message"
+              id="message"
+              required
+              maxlength="500"
+              placeholder=" Enter text here"
+            ></textarea>
+          </div>
+          <div class="form-element btn">
+            <button type="submit" class="sbtn">Get in touch</button>
+          </div>
+        </form>
+      </section>
+      <!--Footer Section-->
+      <footer class="footer">
+        <div class="social-media social-bottom">
+          <a href="https://twitter.com/ndorrh" class="social-links"
+            ><img src="images/icons/twitter.svg" alt="twitter"
+          /></a>
+          <a
+            href="https://www.linkedin.com/in/ndorrh-oswald-ebu-82ab02236/"
+            class="social-links"
+            ><img src="images/icons/Linkedin.svg" alt="linkedin"
+          /></a>
+          <a href="#" class="social-links"
+            ><img src="images/icons/medium.svg" alt="medium"
+          /></a>
+          <a href="https://github.com/ndorrh" class="social-links"
+            ><img src="images/icons/github.svg" alt="github"
+          /></a>
+          <a href="#" class="social-links"
+            ><img src="images/icons/angellist.svg" alt="angellist"
+          /></a>
+        </div>
+        <div>
+          <address class="address-style">myemailhere@email.com</address>
```

Comment:
- [ ] I think it's better to use another tag instead of the `<address>` tag. The `<address>` tag presents your data in `italic` font that it's different from the template. And, also your email address must be a link to send a message to you using your portfolio. As requested by the previous reviewer, please stick to the Figma design as much as possible.